model/testutil/api.go

```diff
@@ -82,7 +86,7 @@ func SetupAPITestData(testConfig *evergreen.Settings, taskDisplayName string, va
 		Branch:      project.Branch,
 		Enabled:     project.Enabled,
 		BatchTime:   project.BatchTime,
-		LocalConfig: string(projectConfig),
```

Comment:
I feel like it's a bug that we store the old projectConfig here and not the new one with the extra build variant. However I also don't really understand why we have this localConfig to begin with, but TestPatchListModulesEndPoints in api_patch_test.go fails without this.